Jenkinsfile

```diff
@@ -581,6 +586,8 @@ pipeline {
                 cmake -GNinja -DNOISEPAGE_UNITY_BUILD=ON -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_BUILD_TYPE=Release -DNOISEPAGE_USE_ASAN=OFF -DNOISEPAGE_USE_JEMALLOC=ON -DNOISEPAGE_BUILD_TESTS=OFF  -DNOISEPAGE_BUILD_SELF_DRIVING_TESTS=ON -DNOISEPAGE_UNITTEST_OUTPUT_ON_FAILURE=ON ..
                 ninja mini_runners''', label: 'Self-driving tests (Compile mini_runners)'
 
+                // mini_runners parameters are arbitrary picked so that it finished within reasonable time (<10min) and
```

Comment:
Sorry, could you elaborate more?

1. Most people do not touch mini_runners, so they do not know what any of these parameters are.
2. Furthermore, if this is just an end-to-end functionality test that does not measure performance, why does it matter how big the data generated is? If the data size does not matter, shouldn't the parameters be picked so that all components are exercised (rather than aiming to hit some arbitrary testing time)?

Template suggestion:

```
// The parameters to the mini_runners target are (arbitrarily picked to complete tests within a reasonable time / picked to exercise all OUs).
// Specifically, the parameters chosen are:
// - mini_runner_rows_limit does (DESCRIPTION). This is set to 1000 because (REASON).
// - rerun does (DESCRIPTION). This is set to 0 because (REASON).
// - warm_num does (DESCRIPTION). This is set to 1 because (REASON).
```

Reply:
Yeah, I think this is why @malin1993ml picked those parameters. I could elaborate more on this.